packages/react-instantsearch-hooks-dom/src/widgets/__tests__/Breadcrumb.test.tsx

```diff
@@ -0,0 +1,375 @@
+              </a>
+            </li>
+            <li
+              class="ais-Breadcrumb-item"
+            >
+              <span
+                aria-hidden="true"
+                class="ais-Breadcrumb-separator"
+              >
+                &gt;
+              </span>
+              <a
+                class="ais-Breadcrumb-link"
+                href="#"
+              >
+                CAMERAS & CAMCORDERS
+              </a>
+            </li>
+            <li
+              class="ais-Breadcrumb-item ais-Breadcrumb-item--selected"
+            >
+              <span
+                aria-hidden="true"
+                class="ais-Breadcrumb-separator"
+              >
+                &gt;
+              </span>
+              DIGITAL CAMERAS
+            </li>
+          </ul>
+        </div>
+      </div>
+    `);
+  });
+
+  test('navigates to a parent category', async () => {
+    const { search, searchClient, hierarchicalAttributes } =
+      getBreadcrumbSearchClient();
+
+    const { container, getByText } = render(
+      <InstantSearchHooksTestWrapper
+        searchClient={searchClient}
+        initialUiState={{
+          indexName: {
+            hierarchicalMenu: {
+              'hierarchicalCategories.lvl0': [
+                'Cameras & Camcorders > Digital Cameras',
+              ],
+            },
+          },
+        }}
+      >
+        <VirtualHierarchicalMenu attributes={hierarchicalAttributes} />
+        <Breadcrumb attributes={hierarchicalAttributes} />
+      </InstantSearchHooksTestWrapper>
+    );
+
+    await wait(0);
+
+    search.mockClear();
+
+    expect(container).toMatchInlineSnapshot(`
+      <div>
+        <div
+          class="ais-Breadcrumb"
+        >
+          <ul
+            class="ais-Breadcrumb-list"
+          >
+            <li
+              class="ais-Breadcrumb-item"
+            >
+              <a
+                class="ais-Breadcrumb-link"
+                href="#"
+              >
+                Home
+              </a>
+            </li>
+            <li
+              class="ais-Breadcrumb-item"
+            >
+              <span
+                aria-hidden="true"
+                class="ais-Breadcrumb-separator"
+              >
+                &gt;
+              </span>
+              <a
+                class="ais-Breadcrumb-link"
+                href="#"
+              >
+                Cameras & Camcorders
+              </a>
+            </li>
+            <li
+              class="ais-Breadcrumb-item ais-Breadcrumb-item--selected"
+            >
+              <span
+                aria-hidden="true"
+                class="ais-Breadcrumb-separator"
+              >
+                &gt;
+              </span>
+              Digital Cameras
+            </li>
+          </ul>
+        </div>
+      </div>
+    `);
+
+    userEvent.click(getByText('Cameras & Camcorders'));
+
+    await wait(0);
+
+    expect(search).toHaveBeenLastCalledWith(
```

Comment:
You can also assert the number of calls before this assertion, so we ensure this doesn't fire more queries than expected.